app/src/test/kotlin/KotlinJunitTest.kt

```diff
@@ -1,17 +1,62 @@
 package test.kotlin
 
+import android.graphics.Matrix
+import android.graphics.Rect
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.verify
+import com.simplemobiletools.camera.models.FocusArea
+import com.simplemobiletools.camera.models.MySize
 import org.junit.Test
 import org.junit.Assert
 
 class KotlinJunitTest {
 
+    // Sample Tests for Travis CI
     @Test
-    fun firstTest() {
-        Assert.assertTrue(1 == 1)
+    fun mySize_modelsTest() {
+        println("Testing class MySize")
+        val mMySize = MySize(16, 9)
+        Assert.assertEquals(9, mMySize.height)
+        Assert.assertEquals(16, mMySize.width)
     }
 
     @Test
-    fun secondTest() {
-        Assert.assertTrue(1 == 1)
+    fun focusArea_modelsTest() {
+        println("Testing class FocusArea")
+        val mFocusArea = FocusArea(Rect(5, 5, 5, 5), 9)
+        Assert.assertEquals(9, mFocusArea.weight)
+    }
+
+    @Test
+    fun mock_test1() {
+        println("Testing Mockito 1")
+        val mockedList: MutableList<Int> = mock()
+        mockedList.add(1)
+        mockedList.clear()
+
```

Comment:
Is there supposed to have a space here with our linter? I'm surprised it did not catch this if yes?

Reply:
I ran the ktlint script and it didn't give any errors, the TravisCI build also passed.

[I think ktlint only complains if you have consecutive blank lines (2 or more).](https://ktlint.github.io/#rule-blank) A blank line is fine in some cases because it can help break up the code into logical "blocks" which can help readability, in my opinion.

Reply:
That is true! My other [commit](https://github.com/aa-software2112/SOEN390_SimpleCamera/pull/31/commits/9725913ee09e5f29d8b208f65d3000adfcef5057) had 2 consecutive blank lines, hence why klint detected it. Thank you for the clarification.